docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.0'
+
+services:
+  rasacore:
+      image: rasa/rasa_core:latest
+      networks: ['rasa-network']
+      ports:
+      - "5005:5005"
+      volumes:
+      - "./rasa-app-data/core-model:/app/model"
```

Comment:
we do have a starter pack to get run from when creating a new bot, can you please make sure the docker compose and that repository play well with one another https://github.com/RasaHQ/starter-pack-rasa-stack